setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import setup, find_packages
+
+
+setup(
+    name='RevPi Device Info',
+    version='1.0',
+    author="Nicolai Buchwitz",
+    author_email="n.buchwitz@kunbus.com",
+    description='RevPi device information from RevPi HAT EEPROM',
+    url='https://revolutionpi.com',
```

Comment:
Is this the correct URL? Wouldn't this be the URL of the git repo?

Reply:
Fixed it